### PR TITLE
Wrong place of "send_key 'alt-n', 2"  when  SCC_ADDONS=sdk  test case

### PR DESCRIPTION
**Pull request by dgutu-suse**

### CODE REVIEW

**Comment by dgutu-suse** [comment]
Old test - http://crocodile.qa.suse.cz/tests/1958

**Comment by dgutu-suse** [comment]
Latest test with changes - http://avicularia.qa.suse.cz/tests/198

File: lib/registration.pm [hunk 1]
```diff
@@ -96,11 +96,11 @@ sub fill_in_registration_data {
                         assert_and_click "scc-code-field-$addon";
                     }
                     type_string $regcode;
+                    send_key 'alt-n';
```
Comment (sysrich) [inline lib/registration.pm]: Surely you at least want to have the alt-n AFTER it's taken the screenshot on line 101?

That screenshot is there so we can be sure it's typed the right Regcode..now we'll never see it..

Comment (dgutu-suse) [inline lib/registration.pm]: Yeap, agree... I will move it to line 101

**Comment by okurz** [comment]
please squash the commits and make sure to use the one as base which has you as author, not "root" :-)

**Comment by dgutu-suse** [comment]
Common, its from my openqa server :)

**Comment by okurz** [comment]
needs rebase and squash

**Comment by okurz** [comment]
LGTM

**Pull request merged**